app/components/channel_drawer/channel_drawer.js

```diff
@@ -125,6 +127,14 @@ export default class ChannelDrawer extends PureComponent {
         };
     };
 
+    handleUpdateTitle = (channel) => {
+        let channelName = '';
+        if (channel.display_name) {
+            channelName = channel.display_name;
+        }
+        this.props.actions.setChannelDisplayName(channelName);
```

Comment:
This will change the display name when other channels are updated, not just the current one

Reply:
The event is only fired when the updsted channel is the current channel.

The websocket does not use the action as this is an action defined in the client and not available in redux

Reply:
See https://github.com/mattermost/mattermost-redux/blob/master/src/actions/websocket.js#L479

Reply:
Alright, I guess that'll work, but I think we need to consider adding a more general purpose mechanism to have the mobile app dispatch actions in response to websocket events. Using the generic event emitter to go outside of redux to have the ChannelDrawer dispatch an event in response feels hacky.